asreview/webapp/src/ProjectComponents/AnalyticsComponents/LabelingFrequency.js

```diff
@@ -94,115 +60,63 @@ const LabelingFrequency = ({ genericDataQuery, progressQuery }) => {
               ? theme.palette.primary.light
               : theme.palette.primary.main // Relevant
             : theme.palette.mode === "light"
-              ? "#808080"
+              ? theme.palette.grey[600]
               : theme.palette.grey[600]; // Irrelevant
 
         ctx.beginPath();
-        const x = canvasWidth - ((index + 1) / totalVisible) * canvasWidth; // Start drawing from right to left
+        const x = canvasWidth - ((index + 1) / totalVisible) * canvasWidth;
         ctx.roundRect(
-          x - lineWidth, // Adjust x to account for line width
-          (fullHeight - lineHeight) / 2, // Center the line vertically
+          x - lineWidth,
+          (fullHeight - lineHeight) / 2,
           lineWidth,
           lineHeight,
-          5, // Rounded corners
+          5,
         );
         ctx.fill();
       });
     }
-  }, [
-    decisionsToDisplay,
-    sliderValue,
-    theme.palette.grey,
-    theme.palette.mode,
-    theme.palette.primary.light,
-    theme.palette.primary.main,
-  ]);
+  }, [decisionsToDisplay, sliderValue, theme]);
 
   const handleSliderChange = (event, newValue) => {
     setSliderValue(newValue);
   };
 
+  const handlePopoverOpen = (event) => {
+    setAnchorEl(event.currentTarget);
+  };
+
+  const handlePopoverClose = () => {
+    setAnchorEl(null);
+  };
+
+  const popoverOpen = Boolean(anchorEl);
+
   return (
-    <StyledCard elevation={2}>
+    <Card
+      sx={{
+        borderRadius: 4,
+        boxShadow: 3,
+        p: 2,
+        maxWidth: 960,
+        width: "100%",
+        position: "relative",
+      }}
```

Comment:
Let's not style components like this. This makes the UI inconsistent and very hard to maintain. If we want to style components, let's do it for all components in the application via https://mui.com/material-ui/customization/theme-components/. 

For this PR, let's remove all styling that doesn't break the UI. At a later stage, we spend some time styling the entire application as described in the docs.